nautobot/core/templates/generic/object_list.html

```diff
@@ -20,14 +37,54 @@
     {% endif %}
 </div>
 <h1>{% block title %}{{ content_type.model_class|meta:"verbose_name_plural"|bettertitle }}{% endblock %}</h1>
+{% if filter_params %}
+<div class="filters-applied">
+    <b>Filters:</b>
+    {% for field, values in filter_params %}
+    <span class="select2 select2-container select2-container--bootstrap display-inline" dir="ltr">
+        <span class="selection">
```

Comment:
What's this span do?

Reply:
@glennmatthews those spans are used conjunction with select2 class. it was cloned from select2